java/src/test/java/com/google/openlocationcode/TestUtils.java

```diff
@@ -0,0 +1,22 @@
+package com.google.openlocationcode;
+
+import java.io.File;
+
+/**
```

Comment:
You can remove the javadocs here. They don't provide any useful info, as the class is pretty self-explanatory.

Reply:
removed